### PR TITLE
Add storybook deploy

### PR DESCRIPTION
**Pull request by iAdramelk**
Should be available at `/storybook/`.

### CODE REVIEW

**Comment by klizhentas** [comment]
@iAdramelk let's make sure `/storybook` is only available on dev view (and linked on the website), not a prod version.

**Comment by iAdramelk** [comment]
> let's make sure /storybook is only available on dev view (and linked here), not a prod version.

@klizhentas can you give me more details? Right now it deploys with the docs site in each branch as `/storybook/`. It will not be visible on the main `goteleport.com` site if you don't add it to redirects manually. Should I remove it from the `main` branch deploy?

**Comment by iAdramelk** [comment]
@klizhentas updated to only deploy for previews.

File: .eslintignore [hunk 1]
```diff
@@ -2,4 +2,5 @@
 /.next
 /node_modules
 /pages/ver/*
-/content
\ No newline at end of file
+/content
+/public/storybook
```
Comment (Saionaro) [inline .eslintignore:6]:
```suggestion
/public/storybook

```

File: .prettierignore [hunk 2]
```diff
@@ -2,4 +2,5 @@
 /.build
 /node_modules
 /pages/ver/*
-/content
\ No newline at end of file
+/content
+/public/storybook
```
Comment (Saionaro) [inline .prettierignore:6]:
```suggestion
/public/storybook

```

**Pull request merged**